draft-ietf-teas-actn-poi-applicability.txt

```diff
@@ -2042,39 +2096,39 @@ Internet-Draft                 ACTN POI                      March 2022
 
    The MDSC translates the L2/L3 VPN SLA requirements into TE
    requirements (e.g., bandwidth, TE metric bounds, SRLG disjointness,
-   nodes/links/domains inclusion/exclusion) and find the SR-TE paths
-   that meet these TE requirements (see section 2.1.1).
+   nodes/links/domains inclusion/exclusion) and find the TE paths that
+   meet these TE requirements (see section 2.1.1).
 
 
 
 
-Peruzzini et al.      Expires September 7, 2022               [Page 38]
+Peruzzini et al.      Expires December 29, 2022               [Page 39]
 
-Internet-Draft                 ACTN POI                      March 2022
+Internet-Draft                 ACTN POI                       June 2022
 
 
    For example, considering the L3VPN in Figure 2, the MDSC finds that:
 
-   o  a PE13-P16-PE14 SR-TE path already exists but have not enough
+   o  a PE13-P16-PE14 TE path already exists but have not enough
       bandwidth to support the new L3VPN, as described in section 4.4;
 
    o  the IP link(s) between P16 and PE14 has not enough bandwidth to
-      support increasing the bandwidth of that SR-TE path, as described
-      in section 4.3;
+      support increasing the bandwidth of that TE path, as described in
+      section 4.3;
 
    o  a new underlay optical tunnel could be setup to increase the
-      bandwidth IP link(s) between P16 and PE14 to support increasing
-      the bandwidth of that overlay SR-TE path, as described in section
-      5.2. The dimensioning of the underlay optical tunnel is decided by
-      the MDSC based on the bandwidth requested by the SR-TE path and on
-      its multi-layer optimization policy, which is an internal MDSC
-      implementation issue.
+      bandwidth of the IP link(s) between P16 and PE14 to support
+      increasing the bandwidth of that overlay TE path, as described in
+      section 5.2. The dimensioning of the underlay optical tunnel is
+      decided by the MDSC based on the bandwidth requested by the TE
+      path and on its multi-layer optimization policy, which is an
+      internal MDSC implementation issue.
 
    Considering for example the L3VPN in Figure 2, the MDSC can also
-   decide that a new multi-domain SR-TE path needs to be setup between
-   PE13 and PE23, e.g., either because existing SR-TE paths between PE13
-   and PE23 are not able to meet the TE and binding requirements of the
-   L2/L3 VPN service or because there is no SR-TE path between PE13 and
+   decide that a new inter-domain TE path needs to be setup between PE13
```

Comment:
maybe cross-domain path should be better: when I think to inter-domain I'm looking to the link between border routers.

Reply:
c/inter-domain TE path/multi-domain TE path/

Reply:
Fixed with https://github.com/FabioPeruzzini/actn-poi/pull/94/commits/db7de1b347f717eebe690317827570fbf6e8970b